apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/BFD-1010-schema-changes/schema_modified_constraints.sql

```diff
@@ -0,0 +1,1931 @@
+/*
+ * This migration has two primary goals:
+ *
+ * First, we want to convert the beneficiaryId and claimId columns to `bigint`, rather than their current
+ * `varchar(15)` types. Per the PostgreSQL documentation, a `varchar(15)` column requires 16 bytes per record
+ * on disk [1], and a `bigint` column only requires 8 bytes. Accordingly, this change roughly halves the size
+ * of our indexes using those columns.
+ *
+ * Second, we're _finally_ moving to a more idiomatic (and user-friendly) naming scheme for these tables based
+ * (mostly) on CCW field names.
+ *
+ * In addition to those primary goals, this schema migration is also being used as a good opportunity to
+ * address all of these other less major schema changes:
+ * 
+ * * TODO
+ *
+ * Overall, these migrations will be done in three steps:
+ *
+ * 1. Clone the tables to new tables with all of the desired schema changes, copying the original data over.
+ * 2. Deploy a new version of the application that only uses those new tables.
+ * 3. Remove the original tables.
+ * 
+ * This script is for Step 1. 
+ *
+ * [1]: https://www.postgresql.org/docs/10/datatype-character.html
+ * [2]: https://www.postgresql.org/docs/10/datatype-numeric.html
+ */
```

Comment:
I'd add file headers to each migration script that explain **all** of the changes being made in them. That'll be very important for anyone trying to figure them out in the future.